backend/graph/api/schema/episodes.graphqls

```diff
@@ -46,6 +46,8 @@ type Episode {
     lessons(first: Int, offset: Int): LessonPagination! @goField(forceResolver: true)
     shareRestriction: ShareRestriction! @goField(forceResolver: true)
     inMyList: Boolean! @goField(forceResolver: true)
+
+    next: [Episode!]! @goField(forceResolver: true)
```

Comment:
should be nullable, we may want to stop at the end of a show or playlist, for example.

Reply:
It will be empty if there's no next episodes.

Reply:
ofc ok